sprint_2/Article.js

```diff
@@ -0,0 +1,44 @@
+class Article {
+  #title;
+  #content;
+  #writer;
+  #likeCount;
+
+  constructor(title, content, writer) {
+    this.#title = title;
+    this.#content = content;
+    this.#writer = writer;
+    this.#likeCount = 0;
+    this.createdAt = new Date();
+  }

```

Comment:
args 검증이 필요한데요.
회사마다 코드컨벤션이 다르고 constructor에서 에러를 던지는 것을 안좋아하는 사람들도 있습니다.
args 검증을 constructor 내부에서 하거나 또는 정적 메서드를 만들어서 하거나 타입스크립트의 경우 타입으로 충분히 검증이 되거나 세 가지 방법정도 있는데 그 중 constructor 내부에서 다음과 같이 검증 해볼 수 있습니다. 

```suggestion
  constructor(title, content, writer) {
      if (!title || typeof title !== "string" || title.trim() === "") {
      throw new Error("제목은 비어있지 않은 문자열이어야 합니다.");
    }
    if (!content || typeof content !== "string" || content.trim() === "") {
      throw new Error("내용은 비어있지 않은 문자열이어야 합니다.");
    }
    if (!writer || typeof writer !== "string" || writer.trim() === "") {
      throw new Error("작성자는 비어있지 않은 문자열이어야 합니다.");
    }
  
    this.#title = title;
    this.#content = content;
    this.#writer = writer;
    this.#likeCount = 0;
    this.createdAt = new Date();
  }

```